staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/helpers.go

```diff
@@ -42,6 +53,7 @@ func NestedField(obj map[string]interface{}, fields ...string) (interface{}, boo
 				return nil, false
 			}
 		} else {
+			glog.Warningf("NestedField(): expected map[string]interface{}, got %T while getting field %q in %s", val, field, fields)
```

Comment:
don't think we want this warning. I would expect that there are ways to trigger this warning from outside of the apiserver, spamming the log.